packages/bits/src/lib/button/button.component.spec.ts

```diff
@@ -107,9 +107,6 @@ describe("components >", () => {
             beforeEach(() => {
                 logger = TestBed.inject(LoggerService);
             });
-            it(`should log an error for buttons without a type attribute`, () => {
-                const errorSpy = spyOnProperty(logger, "error", "get").and.callThrough();
-            });
```

Comment:
It seems like this test was somehow missing the following lines and can be restored:

```
fixture = TestBed.createComponent(TestAppButtonNoTypeComponent);
expect(errorSpy).toHaveBeenCalled();
```

Reply:
firstly I tried do so, but it doesn't works and I decided don't dive more deep in scope my task.

Reply:
For this, it would be better to comment it out and create a ticket for restoring it with the correct expectation. Then, above the commented code put something like:
```
// Restore in scope of NUI-<ticket id>
```

Reply:
sorry, you was right, spec above is valid.